controllers/common/ansible.go

```diff
@@ -5,26 +5,109 @@ package common
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"reflect"
 	"regexp"
+	"sort"
+	"strconv"
 	"strings"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
+	ctrl "sigs.k8s.io/controller-runtime"
 
 	policyv1beta1 "open-cluster-management.io/governance-policy-propagator/api/v1beta1"
 )
 
+const (
+	ControllerName             string = "policy-automation"
+	PolicyAutomationLabel      string = "policyautomation.policy.open-cluster-management.io/policyautomation-name"
+	PolicyAutomationGeneration string = "policyautomation.policy.open-cluster-management.io/policyautomation-generation"
```

Comment:
Sorry to go back and forth on this, but I think leaving this as just the API group might make the most sense and follow previous keys.

```suggestion
	PolicyAutomationLabel      string = "policy.open-cluster-management.io/policyautomation-name"
	PolicyAutomationGeneration string = "policy.open-cluster-management.io/policyautomation-generation"
```